apisix/plugins/limit-count/limit-count-redis-cluster.lua

```diff
@@ -72,62 +71,28 @@ function _M.new(plugin_name, limit, window, conf)
     return setmetatable(self, mt)
 end
 
+local script = "if redis.call('ttl',KEYS[1]) < 0 then "
```

Comment:
Would it be better to use `exists` instead of `ttl`?

Reply:
I think ttl is better. ttl will set the expire time on the key which is permanent. Both are O(1) Time complexity.